azure-test/tests/azure_express_route_circuit/test-not-found-query.sql

```diff
@@ -0,0 +1,3 @@
+select name, akas, tags, title
+from azure.azure_express_route_circuit
+where name = 'dummy-{{resourceName}}' and resource_group = '{{resourceName}}'
```

Comment:
```suggestion
where name = 'dummy-{{ resourceName }}' and resource_group = '{{ resourceName }}'
```
@ParthaI do we need to check both?